### DISCUSSION TITLE
Is this correct notation for adding a condition?

### DISCUSSION THREAD

**Discussion by TheRealAmazonKendra** [Mergifyio/mergify#5045] [General]
I want to add a condition for dismissing reviews that it can't be a review with changes requested by as specific user. Is the line below correct notation? 

`- "#changes-requested-reviews-by!=such-and-such"`

**Comment by jd** [comment]
If `such-and-such` is a user, you should be:
```
- "changes-requested-reviews-by!=such-and-such"
```

The `#` operator is only for length of the list.
